easybuild/easyconfigs/s/snakemake/snakemake-7.18.1-foss-2020a.eb

```
@@ -0,0 +1,96 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '7.18.1'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2'),
+    ('IPython', '7.15.0', '-Python-3.8.2'),
+    ('poetry', '1.0.9', '-Python-3.8.2'),
+    ('GitPython', '3.1.9', '-Python-3.8.2'),
+]
```

if you're depending on easyconfigs with a `Python` versionsuffix, then this easyconfig should also have that same suffix

```suggestion
name = 'snakemake'
version = '7.18.1'
versionsuffix = '-Python-%(pyver)s'

homepage = 'https://snakemake.readthedocs.io'
description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."

toolchain = {'name': 'foss', 'version': '2020a'}

dependencies = [
    ('Python', '3.8.2'),
    ('SciPy-bundle', '2020.03', versionsuffix),
    ('IPython', '7.15.0', versionsuffix),
    ('poetry', '1.0.9', versionsuffix),
    ('GitPython', '3.1.9', versionsuffix),
]
```

---

This makes sense, I did --update-pr with updated eb file.

---

Ok, done. I'm still learning how all this works ... thanks for patience :)